nbd_http_server.py

```diff
@@ -274,10 +280,17 @@ def clean_paths():
 
     # Continue to log server messages in stdout.
     def log_server_messages():
+        verbose_keywords = [
+            'pread',
+            'pwrite'
+        ]
```

Comment:
You can use a tuple here.

Reply:
fixed in [69f9239](https://github.com/xcp-ng/http-nbd-transfer/pull/14/commits/69f9239aaf9647afb4db1454fa878fa28fcd9896)